src/themes/oneui/mixins/colors.scss

```diff
@@ -1,4 +1,4 @@
-$contexts: (neutral, brand, primary, accent, info, good, warning, bad);
+$contexts: (neutral, primary, info, success, warning, danger);
```

Comment:
This is different from the change in `src/constants/style-related.ts`. I think this one is better, but maybe we should discuss. This actually effects a lot of the changes in consequence, so perhaps we should get to a conclusion about this before doing a lot of refactoring.

Reply:
we decided to leave 3 types of context: success, warning, danger
and add other context types only for test